RecoPixelVertexing/PixelTrackFitting/interface/RiemannFit.h

```diff
@@ -515,7 +515,7 @@ namespace riemannFit {
     // scale
     const double tempQ = mc.squaredNorm();
     const double tempS = sqrt(n * 1. / tempQ);  // scaling factor
-    p3D *= tempS;
+    p3D.block(0, 0, 2, n) *= tempS;
```

Comment:
was the older code running, BTW?
If I understood the warning correctly, was there supposed to be a crash here before this fix?

Reply:
No, we just scaled uninitialized values in place in the last row of the matrix just to overwrite them on the next statement. It was totally harmless (besides wasting `n` multiplications, and being the last compilation warning to clear).